bloom-instance/tfvars.template

```diff
@@ -211,3 +211,44 @@ partner_site = {
     health_check = {}
   }
 }
+
+backend_api = {
+  name = "backend"
+  port = 3100
+
+  task = {
+    cpu   = 256
+    ram   = 512
+    image = "<repo>/backend:run"
+
+    env_vars = {
```

Comment:
I'm curious if we want to move any application level settings into the application repo in the medium/long term. What's the vision for deploying the application with a new env var setting- is it editing this repo and doing a tf run, then triggering a cicd run?

Reply:
The goal is to have all env vars set in the runtime environment rather than during build time.  This way, we can easily change vars between dev, staging, prod, etc without having to rebuild a new image for each one.  Changing an env var (which should be pretty rare) would be done by adding it to the tfvars file and redeploying via `terraform apply`. The config files will live in this repo and all infra changes will be managed via an infra pipeline, so from a developer standpoint it just looks like making a branch, changing the file, pushing the commit up, and getting it approved and merged.

Reply:
By infra pipeline, do you mean automation that runs `terraform apply`? I was under the impression the cicd pipeline would be the place to define application level runtime env vars.

Reply:
Yes, I mean the pipeline that runs `terraform apply`.  Runtime env vars are set on the task definition, not the build process